controlplane/migrations/0112_fluffy_marvel_apes.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE IF NOT EXISTS "namespace_cache_warmer_config" (
```

Comment:
Wouldn't it be a good idea to leverage this table for general namespace settings?